modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -3691,6 +3694,22 @@ private void _updateTable(
 			}
 
 			if (!values.containsKey(objectField.getName())) {
+				if (objectField.isRequired() &&
+					(workflowAction != WorkflowConstants.ACTION_SAVE_DRAFT)) {
+
+					ObjectEntry objectEntry = fetchObjectEntry(objectEntryId);
+
+					if ((objectEntry != null) &&
+						Validator.isNull(
+							MapUtil.getString(
+								objectEntry.getValues(),
+								objectField.getName()))) {
```

Comment:
Why do we need to check this map if we have this condition i`f (!values.containsKey(objectField.getName()))` just before?

Reply:
No changes.